server/services/GameLogic.js

```diff
@@ -239,6 +247,12 @@ module.exports = class GameLogic {
     /* Returns true if all players (apart from the current player) have voted this round */
     allPlayersVoted() { return this.players.every(player => player.username === this.currentPlayer.username || this.playerHasVoted(player.username)) };
 
+    /* Mark the player's turn as finished and notify the other players */
+    markTurnAsFinished() {
+        this.players.find(player => player.username === username).finishedTurn = true;
```

Comment:
Where is `username` defined? Is it a global variable?

Reply:
oops, that's when I moved it to a function, that's why the tests aren't passing, my bad